docs/qaoa/routing_with_tket.ipynb

```diff
@@ -401,12 +367,11 @@
       },
       "outputs": [],
       "source": [
-        "def tk_to_i(tk):\n",
-        "    i = tk.index\n",
-        "    assert len(i) == 1, i\n",
-        "    return i[0]\n",
+        "def tk_to_cirq_qubit(tk):\n",
+        "    ind = tk.index\n",
+        "    return cirq.LineQubit(ind[0]) if len(ind) == 1 else cirq.GridQid(*ind, dimension=len(ind))\n",
```

Comment:
the `dimension` argument being equal to `len(ind)` is a coincidence and should be omitted. This is to support qudits. Just use `cirq.GridQubit`.